browser/ui/views/toolbar/brave_browser_app_menu_button.cc

```diff
@@ -37,5 +48,10 @@ bool BraveBrowserAppMenuButton::ShouldBlendHighlightColor() const {
   return true;
 }
 
+// Same Insest for W/O Label.
+void BraveBrowserAppMenuButton::UpdateLayoutInsets() {
+  SetLayoutInsets(::GetLayoutInsets(TOOLBAR_BUTTON));
```

Comment:
can you add test code to `BraveAppMenuBrowserTest.AppMenuButtonUpgradeAlertTest` for checking our insets value?

Reply:
Sure! Will add it.